hack/multi-node/README.md

```diff
@@ -10,6 +10,22 @@ This will generate the default assets in the `cluster` directory and launch mult
 ./bootkube-up
 ```
 
+Configure Kubeconfig env variable allow using kubectl without paramaters
```

Comment:
nit 1: I would add `(optional)` to this step. It is not strictly required.
nit 2: "... env variable to allow"
nit 3: s/paramaters/parameters